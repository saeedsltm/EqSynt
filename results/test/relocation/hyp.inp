select_weighted.out
n
