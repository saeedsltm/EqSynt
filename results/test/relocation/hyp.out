 2020  1 1 0001 40.2 L  35.967  51.792 22.6  New  8 0.3                        1
 GAP=224        0.50       1.9     4.4  4.2 -0.9025E+00 -0.5907E+01  0.2397E+01E
 Action:ARG 22-06-13 19:38 OP:OBSP STATUS:               ID:20200101000140     I
 STAT SP IPHASW D HRMM SECON CODA AMPLIT PERI AZIMU VELO AIN AR TRES W  DIS CAZ7
 VGNX    EP   3    0 147.910                             116    0.40 3 31.3  40 
 VGNX    ES   2    0 152.968                             116   -0.05 5 31.3  40 
 YOPU    EP   1    0 156.347                              62   -0.12 8 88.6 150 
 YOPU    ES   4    0 2 6.959                              62   -1.74 0 88.6 150 
 HSDK    EP   0    0 156.901                              62    0.0110 91.5  34 
 HSDK    ES   4    0 211.782                              62    2.35 0 91.5  34 
 TYAW    EP   2    0 157.757                              62    0.84 5 91.6 138 
 TYAW    ES   1    0 2 9.347                              62   -0.13 7 91.6 138 
 PAMN    EP   0    0 159.586                              62    0.0610  109  15 
 PAMN    ES   4    0 212.800                              62   -1.25 0  109  15 
 CIDP    EP   1    0 2 0.822                              62   -0.43 7  121  50 
 CIDP    ES   4    0 216.202                              62   -0.86 0  121  50 
 OIXZ    EP   4    0 2 2.403                              62   -1.87 0  142  44 
 OIXZ    ES   4    0 221.422                              62   -0.93 0  142  44 
 ESZY    EP   3    0 2 9.191                              62   -0.78 2  180  58 
 ESZY    ES   0    0 232.482                              62    0.16 8  180  58 
 MZGD    EP   4    0 211.875                              62    1.85 0  181  60 
 MZGD    ES   0    0 232.411                              62   -0.00 8  181  60 
                                                                                
 2020  1 1 0003 19.2 L  36.857  50.891 18.1  New  5 0.2                        1
 GAP=320        0.43      19.2    73.6109.3 -0.1276E+04 -0.7998E+04  0.1819E+04E
 Action:ARG 22-06-13 19:38 OP:OBSP STATUS:               ID:20200101000320     I
 STAT SP IPHASW D HRMM SECON CODA AMPLIT PERI AZIMU VELO AIN AR TRES W  DIS CAZ7
 PAMN    EP   4    0 336.823                              62   -1.87 0  108  86 
 PAMN    ES   0    0 353.293                              62   -0.0410  108  86 
 VGNX    EP   0    0 341.219                              62    0.0110  125 126 
 VGNX    ES   4    0 355.869                              62   -1.87 0  125 126 
 CIDP    EP   1    0 348.121                              62   -0.38 6  175  97 
 CIDP    ES   1    0 410.554                              62    0.06 6  175  97 
 ESZY    EP   4    0 356.341                              62   -0.88 0  234  91 
 ESZY    ES   3    0 426.306                              62    0.56 2  234  91 
 MZGD    EP   0    0 358.081                              62    0.32 7  238  92 
 MZGD    ES   4    0 424.817                              62   -1.87 0  238  92 
                                                                                
 2020  1 1 0005  0.2 L  35.552  50.984 25.4  New  3 0.4                        1
 GAP=314        0.42      22.0    42.1999.9  0.7359E+03  0.5942E+06  0.2500E+06E
 Action:ARG 22-06-13 19:38 OP:OBSP STATUS:               ID:20200101000500     I
 STAT SP IPHASW D HRMM SECON CODA AMPLIT PERI AZIMU VELO AIN AR TRES W  DIS CAZ7
 VGNX    EP   1    0 520.319                              62   -0.08 7  116  53 
 VGNX    ES   3    0 535.537                              62    0.00 2  116  53 
 TYAW    EP   4    0 522.601                              62   -0.69 0  136  99 
 TYAW    ES   3    0 540.591                              62   -0.00 2  136  99 
 CIDP    EP   3    0 534.893                              62    1.18 2  207  53 
 CIDP    ES   4    0 558.199                              62   -0.64 0  207  53 
                                                                                
 2020  1 1 0006 40.0 L  35.357  52.227 10.0  New  6 0.3                        1
 GAP=192        0.55      20.1    13.5 20.7  0.2506E+03 -0.2553E+03 -0.4159E+03E
 Action:ARG 22-06-13 19:38 OP:OBSP STATUS:               ID:20200101000640     I
 STAT SP IPHASW D HRMM SECON CODA AMPLIT PERI AZIMU VELO AIN AR TRES W  DIS CAZ7
 YOPU    EP   0    0 642.812                             132   -0.1910 10.4 156 
 YOPU    ES   2    0 645.725                             132    0.47 5 10.4 156 
 VGNX    EP   1    0 658.019                              56   -0.14 7 93.7 348 
 VGNX    ES   1    0 711.730                              56   -0.06 7 93.7 348 
 HSDK    EP   4    0 7 6.990                              47    1.26 0  144   5 
 HSDK    ES   0    0 725.231                              47    0.20 8  144   5 
 AURM    EP   4    0 710.708                              47    1.96 0  165  23 
 AURM    ES   0    0 730.573                              47    0.25 8  165  23 
 PAMN    EP   2    0 710.871                              47    0.77 4  174 356 
 PAMN    ES   3    0 732.022                              47   -0.66 2  174 356 
 MZGD    EP   1    0 713.023                              47   -0.34 5  196  37 
 MZGD    ES   1    0 737.993                              47   -0.40 5  196  37 
                                                                                
 2020  1 1 0008 20.5 L  35.634  51.596  2.8  New  6 0.4                        1
 GAP=323        0.46      59.8    51.7 60.4  0.3045E+04 -0.3092E+04 -0.3597E+04E
 Action:ARG 22-06-13 19:38 OP:OBSP STATUS:               ID:20200101000820     I
 STAT SP IPHASW D HRMM SECON CODA AMPLIT PERI AZIMU VELO AIN AR TRES W  DIS CAZ7
 PAMN    EP   3    0 848.555                              41    0.47 3  150  17 
 PAMN    ES   1    0 9 8.761                              41   -0.00 8  150  17 
 CIDP    EP   4    0 850.868                              41    1.40 0  159  44 
 CIDP    ES   3    0 910.399                              41   -0.80 3  159  44 
 AURM    EP   0    0 851.288                              41    0.0610  171  45 
 AURM    ES   1    0 914.783                              41    0.51 7  171  45 
 OIXZ    EP   0    0 852.549                              41   -0.1210  181  40 
 OIXZ    ES   2    0 916.135                              41   -0.65 5  181  40 
 MZGD    EP   3    0 856.822                              41   -0.95 2  216  54 
 MZGD    ES   0    0 925.622                              41   -0.10 9  216  54 
 ESZY    EP   2    0 858.159                              41    0.34 4  216  52 
 ESZY    ES   4    0 928.119                              41    2.32 0  216  52 
                                                                                
 2020  1 1 0010  0.3 L  37.143  52.275 19.8  New  7 0.4                        1
 GAP=293        0.61       4.7     3.5  4.8 -0.5156E+01  0.2508E+01 -0.5803E+01E
 Action:ARG 22-06-13 19:38 OP:OBSP STATUS:               ID:20200101001000     I
 STAT SP IPHASW D HRMM SECON CODA AMPLIT PERI AZIMU VELO AIN AR TRES W  DIS CAZ7
 HSDK    EP   0    01011.416                              94    0.0810 55.0 172 
 HSDK    ES   2    01019.179                              94   -0.46 5 55.0 172 
 OIXZ    EP   2    01013.213                              93    0.59 5 62.7 118 
 OIXZ    ES   0    01021.964                              93    0.0710 62.7 118 
 CIDP    EP   0    01014.313                              92   -0.0310 73.0 137 
 CIDP    ES   2    01024.363                              92   -0.54 5 73.0 137 
 AURM    EP   2    01013.968                              92   -0.83 5 75.8 128 
 AURM    ES   0    01025.885                              92    0.1810 75.8 128 
 ESZY    EP   4    01019.682                              62   -1.19 0  116 108 
 ESZY    ES   1    01036.290                              62   -0.04 7  116 108 
 MZGD    EP   1    01021.265                              62   -0.34 7  121 110 
 MZGD    ES   4    01040.121                              62    2.50 0  121 110 
 TYAW    EP   1    01033.642                              62    0.50 5  200 175 
 TYAW    ES   1    01057.889                              62    0.09 5  200 175 
                                                                                
 2020  1 1 0011 39.2 L  35.387  51.863 37.5  New  4 0.3                        1
 GAP=324        0.41       8.3    34.5 19.8 -0.2048E+03 -0.6621E+03  0.1353E+03E
 Action:ARG 22-06-13 19:38 OP:OBSP STATUS:               ID:20200101001140     I
 STAT SP IPHASW D HRMM SECON CODA AMPLIT PERI AZIMU VELO AIN AR TRES W  DIS CAZ7
 VGNX    EP   2    01155.205                             102   -0.35 5 89.5   9 
 VGNX    ES   0    012 7.820                             102    0.0110 89.5   9 
 HSDK    EP   3    012 4.993                              96    1.02 2  147  18 
 HSDK    ES   4    01223.372                              96    0.83 0  147  18 
 PAMN    EP   4    012 9.686                              95    2.16 0  172   7 
 PAMN    ES   1    01228.814                              95    0.06 6  172   7 
 ESZY    EP   0    01213.612                              57    0.10 7  217  43 
 ESZY    ES   0    01239.165                              57   -0.07 7  217  43 
                                                                                
 2020  1 1 0013 20.0 L  35.719  52.284  4.0  New  7 0.3                        1
 GAP=154        0.41       1.4     4.1  4.2  0.3506E+01 -0.1455E+02 -0.4209E+01E
 Action:ARG 22-06-13 19:38 OP:OBSP STATUS:               ID:20200101001320     I
 STAT SP IPHASW D HRMM SECON CODA AMPLIT PERI AZIMU VELO AIN AR TRES W  DIS CAZ7
 YOPU    EP   4    01330.950                              64    0.80 0 49.6 181 
 YOPU    ES   1    01337.642                              64   -0.09 9 49.6 181 
 VGNX    EP   3    01330.917                              64   -0.75 3 57.1 335 
 VGNX    ES   1    01340.195                              64   -0.18 9 57.1 335 
 HSDK    EP   4    01338.484                              49   -1.77 0  104   4 
 HSDK    ES   3    01356.120                              49    0.71 3  104   4 
 AURM    EP   4    01341.804                              41   -1.82 0  126  28 
 AURM    ES   4    01359.375                              41   -1.93 0  126  28 
 PAMN    EP   2    01345.124                              41    0.30 5  134 353 
 PAMN    ES   1    014 3.580                              41    0.17 8  134 353 
 OIXZ    EP   4    01346.551                              41    0.90 0  140  23 
 OIXZ    ES   0    014 4.900                              41    0.0510  140  23 
 MZGD    EP   1    01349.091                              41    0.17 7  162  44 
 MZGD    ES   0    01410.329                              41   -0.24 9  162  44 
 ESZY    EP   1    01349.389                              41    0.27 7  164  42 
 ESZY    ES   0    01410.925                              41   -0.01 9  164  42 
                                                                                
 2020  1 1 0015  0.3 L  36.596  51.977 16.7  New  6 0.4                        1
 GAP=253        0.55       5.7     9.4  6.6 -0.4813E+02 -0.3954E+02  0.1992E+02E
 Action:ARG 22-06-13 19:38 OP:OBSP STATUS:               ID:20200101001500     I
 STAT SP IPHASW D HRMM SECON CODA AMPLIT PERI AZIMU VELO AIN AR TRES W  DIS CAZ7
 VGNX    EP   2    015 9.946                             104    0.02 5 45.8 176 
 VGNX    ES   4    01517.997                             104    0.89 0 45.8 176 
 CIDP    EP   4    01513.715                              64   -1.44 0 76.9  84 
 CIDP    ES   1    01526.302                              64    0.03 8 76.9  84 
 OIXZ    EP   2    01517.427                              64    0.41 5 88.0  69 
 OIXZ    ES   0    01529.299                              64   -0.2210 88.0  69 
 ESZY    EP   2    01524.983                              53    0.38 4  139  80 
 ESZY    ES   1    01543.029                              53    0.23 7  139  80 
 TYAW    EP   2    01525.844                              53    0.37 4  145 162 
 TYAW    ES   1    01544.476                              53    0.16 6  145 162 
 YOPU    EP   2    01525.124                              53   -0.95 4  149 170 
 YOPU    ES   3    01544.323                              53   -1.06 2  149 170 
                                                                                
 2020  1 1 0016 40.3 L  35.995  51.735  7.0  New  6 0.4                        1
 GAP=249        0.58       1.9    17.8  9.5 -0.1897E+02 -0.1455E+03  0.9503E+01E
 Action:ARG 22-06-13 19:38 OP:OBSP STATUS:               ID:20200101001640     I
 STAT SP IPHASW D HRMM SECON CODA AMPLIT PERI AZIMU VELO AIN AR TRES W  DIS CAZ7
 VGNX    EP   4    01647.258                              97   -0.02 0 32.7  50 
 VGNX    ES   4    01656.800                              97    4.30 0 32.7  50 
 HSDK    EP   0    01658.558                              56    0.0610 91.9  37 
 HSDK    ES   2    01712.815                              56    0.67 5 91.9  37 
 YOPU    EP   1    01659.131                              56    0.30 7 93.9 149 
 YOPU    ES   0    01712.828                              56    0.1010 93.9 149 
 TYAW    EP   1    01658.978                              56   -0.44 7 97.4 137 
 TYAW    ES   4    01712.549                              56   -1.20 0 97.4 137 
 PAMN    EP   4    017 1.419                              47    0.31 0  108  18 
 PAMN    ES   4    01717.547                              47    0.84 0  108  18 
 CIDP    EP   3    017 4.229                              47    0.86 2  123  53 
 CIDP    ES   1    01720.337                              47   -0.32 7  123  53 
 AURM    EP   4    017 6.720                              47    1.57 0  135  53 
 AURM    ES   4    01722.323                              47   -1.46 0  135  53 
 OIXZ    EP   0    017 6.205                              47   -0.09 9  143  46 
 OIXZ    ES   1    01725.862                              47    0.08 7  143  46 
 ESZY    EP   4    01713.524                              47    1.33 0  183  60 
 ESZY    ES   4    01737.775                              47    1.67 0  183  60 
 MZGD    EP   2    01711.602                              47   -0.67 4  184  62 
 MZGD    ES   4    01734.832                              47   -1.42 0  184  62 
                                                                                
